src/views/v2/top-bar/actions.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * View: Top Bar - Actions
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/top-bar/actions.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+?>
+<div class="tribe-events-c-top-bar__actions">
+	<?php $this->template( 'top-bar/actions/hide-recurring' ); ?>
```

Comment:
The idea is that this piece was moved to PRO as recurring events is only a PRO functionality.

Reply:
The hide recurring file in TEC is blank. It hooks in from PRO.

Reply:
Yep, that's what I mean, shouldn't we remove completely the hidden file and hook the PRO template at another point? I mean, it's the only action of the `top-bar__actions`.

Reply:
We want to make sure we have an empty content file for the actions bar as a whole, not specific to hide recurring.

Reply:
done, changed back to actions.